core/lib/rom/schema.rb

```diff
@@ -131,6 +131,8 @@ def self.attributes(attributes, attr_class)
     def initialize(*)
       super
 
+      initialize_primary_key_names
```

Comment:
why was this added?

Reply:
@solnic looks like after `schema.with(relations: registry)` the schema it returns have not initialized `primary_keys_names` and we were getting some errors in different parts of the specs, maybe the problem does not lie in the schema initialize method but in another part of the code.

```ruby
ROM::SQL::MissingPrimaryKeyError:
        Missing primary key for :users
```
Once I added that line the build looks like its working, but maybe there is something else that I'm missing. I happy will keep looking for other solution.